gridstatus/caiso.py

```diff
@@ -1484,6 +1502,13 @@ def _get_historical(
     # but has nulls for all other columns
     df = df.dropna(subset=df.columns[1:], how="all")
 
+    # sometimes this data is from the previous day, in which case we want to label it as such
+    latest_file_time = check_latest_value_time(df, file)
+    current_caiso_time = pd.Timestamp.now(tz="US/Pacific")
```

Comment:
Nit: `tz=self.default_timezone` to avoid using the string.

Reply:
makes sense, though `_get_historical()` is not a class method with a self param and is not currently being passed the timezone. 

see "timezone" params just below it doing a similar str reference. 

```
df["Time"] = df["Time"].apply(
        _make_timestamp,
        today=date,
        timezone="US/Pacific",
    )
```

Reply:
Ah, should I do `CAISO.default_timezone`?

Reply:
Pushed, lmk what you think.

Reply:
> though _get_historical() is not a class method

My fault. Looks like you got it.